api/envoy/config/listener/v3/quic_config.proto

```diff
@@ -77,4 +77,8 @@ message QuicProtocolOptions {
   // [#extension-category: envoy.quic.server_preferred_address]
   core.v3.TypedExtensionConfig server_preferred_address_config = 9
       [(xds.annotations.v3.field_status).work_in_progress = true];
+
+  // Configure the server to send transport parameter :ref:`disable_active_migration <https://www.rfc-editor.org/rfc/rfc9000#section-18.2-4.30.1>`.
+  // Defaults to false (do not send this transport parameter).
+  google.protobuf.BoolValue send_disable_active_migration = 10;
```

Comment:
I wasn't sure if this should be bool or BoolValue; I think there's a non-zero (but low) chance we could decide to change the default at some point.

Reply:
Yep, I just linked to the RFC for docs because the behavior is subtle and I didn't want to make promises that aren't realistic.